packages/openactive-integration-tests/test/features/core/simple-book-free-opportunities/implemented/with-erroneous-payment-property-test.js

```diff
@@ -0,0 +1,74 @@
+const chai = require('chai');
+const chakram = require('chakram');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { FlowHelper } = require('../../../../helpers/flow-helper');
+const { RequestState } = require('../../../../helpers/request-state');
+const { GetMatch, C1, C2, B } = require('../../../../shared-behaviours');
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'core',
+  testFeature: 'simple-book-free-opportunities',
+  testFeatureImplemented: true,
+  testIdentifier: 'with-erroneous-payment-property',
+  testName: 'Fail free bookings which include erroneous payment property',
+  testDescription: 'C1, C2 and B with payment property: payment property is provided but not expected in the request, so an UnnecessaryPaymentDetailsError must be returned.',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookableFree',
+  // The secondary opportunity criteria to use for multiple OrderItem tests
+  // This must also be TestOpportunityBookableFree as the payment property is only disallowed if ALL items are free.
+  controlOpportunityCriteria: 'TestOpportunityBookableFree',
```

Comment:
I just want to confirm that this makes sense. My theory so far is that the "control" is so that the "multiple order items" test has some normal opportunities as well as the specialised opportunities relevant to the test.

TODO

Reply:
Yep that's right, and makes sense